### ISSUE TITLE
Allow to pass authorization token directly

### ISSUE THREAD

**Issue by FredyC** [Urigo/graphql-cli#283]
Just saw this and my eyes got that annoying look. Why so ugly? :)

```bash
yarn graphql get-schema --header "{\"Authorization\": \"Bearer $TOKEN\"}"
```

It would be lovely to accept this comment authorization schema and be able to do just.

```bash
yarn graphql get-schema --token $TOKEN
```

**Comment by marktani** [comment]
This works:

```sh
graphql get-schema --project db --header Authorization="Bearer $TOKEN"
```

**Comment by FredyC** [comment]
Yea keep in mind that this "nice way" works in the command line, but not if you want to have in eg. npm scripts, then you need to escape and it's ugly. With a direct argument there would be less escaping :)

**Comment by moxious** [comment]
This would be a very valuable feature to have.  For example, there are various providers that can deploy APIs behind an OAuth2 layer, and without the ability to use `--header`, these APIs can't really be accessed by `graphql-cli`.

I'm coming to this issue late using version 3.0.11.   The above advice about the --header flag appears not to work.  Some commands will complain that this is an unrecognized flag.  Others will accept the flag, but they seem not to respect it or pass it through to the server.

**Comment by ardatan** [comment]
Currently we have new structure for GraphQL CLI and in the new alpha version `4.0.0-alpha.0`.
Headers can be defined inside [GraphQL Config](https://graphql-config.com/docs/schema) like below;
```yml
schema:
  - URL:
      headers:
         Authorization: [TOKEN]
```
Checkout new instructions and we will be happy to hear your feedback. Feel free to contact us using our Discord channel;
https://discord.gg/xud7bH

**Comment by Urigo** [comment]
We've just released a new version and an alpha release channel for GraphQL CLI - `4.0.0-alpha.XXX`!

We've also moved to use the new `alpha` version of [`graphql-config`](https://medium.com/the-guild/graphql-config-708dd0e5d15f)!

Headers can be defined inside [GraphQL Config](https://graphql-config.com/docs/schema) like below;
```yml
schema:
  - URL:
      headers:
         Authorization: [TOKEN]
```

Would that work to solve your need?

Checkout new instructions and the [migration guide](https://github.com/urigo/graphql-cli#migration-from-3xx-to-4xx) and we will be happy to hear your feedback during the alpha period!

Feel free to contact us by opening an issue or using our Discord channel;
https://discord.gg/xud7bH
